guides/keras_nlp/neural_machine_translation.py

```diff
@@ -0,0 +1,442 @@
+"""
+Title: English-to-Spanish translation with KerasNLP
+Author: [Abheesht Sharma](https://github.com/abheesht17/)
+Date created: 2022/05/26
+Last modified: 2022/05/26
+Description: Use KerasNLP to train a sequence-to-sequence Transformer model on the machine translation task.
+"""
+
+"""
+## Introduction
```

Comment:
Gotta credit the original example. You're using a lot of text copy from the original. It's also a useful reference for readers to see where they can see the "everything from scratch" workflow.